src/styles/index.scss

```diff
@@ -2,2 +1,209 @@
+  color: rgba(255, 255, 255, 1);
+  background-color: $blue-accent;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border-radius: $rounding;
+  transition: $transition-duration;
+
+  &:hover {
+    background-color: $white;
+    color: $blue-accent;
+    border: $blue-border;
+  }
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 200px));
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  border: $gray-border;
+  padding: 32px 16px 16px;
+  border-radius: $rounding;
+  transition: $transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    margin-bottom: 40px;
+    padding-inline: 3px;
+    min-height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover .card__title {
+    color: $title-hover;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $secondary;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-size: 10px;
+    color: $main-accent;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    line-height: 18px;
+    color: $secondary;
+  }
+
+  &__price-number {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-accent;
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline: 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__logo-link {
+    display: flex;
+  }
+}
+
+.nav {
+  text-transform: uppercase;
+  font-weight: 500;
+
+  &__list {
+    display: flex;
+    list-style: none;
+
+    & :not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    position: relative;
+    display: block;
+    text-decoration: none;
+    color: $main-accent;
+    padding-block: 23px;
+    transition: $transition-duration;
+
+    &--hover:hover {
+      color: $blue-accent;
+    }
+
+    &--active {
+      color: $blue-accent;
+
+      &::before {
+        content: '';
+        display: block;
+        position: absolute;
+        left: 0;
+        bottom: 0;
+        width: 100%;
+        height: 4px;
+        border-radius: 8px;
+        background-color: $blue-accent;
+      }
+    }
+  }
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14px;
+  color: $main-accent;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+
+  & :not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--active :not(:last-child) {
+    background-image: url(/src/images/star-active.svg);
```

Comment:
The 'background-image' URL for '.stars--active' is set to '/src/images/star-active.svg'. Ensure that the path is correct and the image is available at the specified location.